### DISCUSSION TITLE
Can Luvi UI be used along side Pines UI

### DISCUSSION THREAD

**Discussion by ClosetGeek-Git** [luvi-ui/laravel-luvi#3] [General]
I realize that this is a complete noob question, but I haven't used Laravel in years and I'm starting to consider the TALL stack. My main question is if components made from Alpine/Tailwind based UI's are interchangeable without a lot of effort fixing css conflicts etc

**Comment by anotherfrontendguy** [comment]
Hey ✌️
You can use Luvi UI and Pines UI inside the same project, however I would not recommend mixing different UI styles inside one project.

I'm not sure if this answers your question though 😅 Do you have an example of what you mean?
